### PR TITLE
Fixes #9422: Update documentation to have version independant repo

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9422

### CODE REVIEW

File: 10_installation/10_install_server/13_install_root_server_centos_rhel.txt [hunk 1]
```diff
@@ -11,9 +11,9 @@ Then run the following command as root:
 version=$(grep "release [0-9]" /etc/redhat-release | sed "s/^.*release \([0-9]\).*$/\1/")
```
Comment (Kegeruneku) [inline 10_installation/10_install_server/13_install_root_server_centos_rhel.txt]: This is now useless :)

File: 10_installation/11_install_agent/20_install_agent_centos_rhel.txt [hunk 2]
```diff
@@ -9,9 +9,9 @@ Then define a yum repository for Rudder:
 version=$(grep "release [0-9]" /etc/redhat-release | sed "s/^.*release \([0-9]\).*$/\1/")
```
Comment (Kegeruneku) [inline 10_installation/11_install_agent/20_install_agent_centos_rhel.txt]: likewise

**Comment by amousset** [comment]
Commit modified

File: 10_installation/10_install_server/13_install_root_server_centos_rhel.txt (hunk 1 above)
Comment (amousset) [inline 10_installation/10_install_server/13_install_root_server_centos_rhel.txt]: Indeed :)

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**